python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_node_type.schema.yml

```
@@ -916,6 +916,9 @@ $defs:
               table: node-type-l2-mlag-configuration
             type: bool
             description: Enable MST PVST border ports.
+          spanning_tree_port_id_allocation_port_channel_range:
+            type: dict
```

Same table as other spanning-tree settings.
```suggestion
            type: dict
            documentation_options:
              table: node-type-l2-mlag-configuration
```

---

updated